### DISCUSSION TITLE
[v1.0.0b] - Data Management Recipe UI should allow clicking recipes

### DISCUSSION THREAD

**Discussion by epheterson** [mealie-recipes/mealie#1454] [Feature Request]
### First Check

- [X] This is not a feature request
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Mealie documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.

### What is the issue you are experiencing?

Recipes should be clickable in Manage Data UI.

The Manage Data UI is a nice way to get an overview of recipes, but if you come across a single one you want to change quickly, you have to go find it in another way or use the bulk editor to change it.

It'd be nice if you could simply click the recipe title and get to the recipe from the Manage Data UI.

### Deployment

Docker (Synology)

### Deployment Details

```
**Details**
Version: v1.0.0beta-3
Build: 56eb0bca7142715984c8028c4c881f086e979564
Application Mode: Production
Demo Status: Not Demo
API Port: 9000
API Docs: Enabled
Database Type: sqlite
Recipe Scraper Version: 14.3.1

**Checks**
Secure Site: Yes
Server Side Base URL: Yes
LDAP Ready: No
Email Configured: No
Docker Volumes: success
```

**Comment by michael-genson** [comment]
This was added at some point, then it broke at some point, but will be fixed by https://github.com/mealie-recipes/mealie/pull/3970
